spec/requests/swagger_docs/v5/partner_financials_spec.rb

```diff
@@ -39,219 +39,34 @@
                       description: "One or more additional properties owned by the applicant's partner",
-                      items: {
-                        type: :object,
-                        required: %i[value outstanding_mortgage percentage_owned shared_with_housing_assoc],
-                        description: "Additional property details",
-                        properties: {
-                          value: {
-                            "$ref" => "#/components/schemas/currency",
-                            description: "Financial value of the property",
-                            example: 500_000.01,
-                          },
-                          outstanding_mortgage: {
-                            "$ref" => "#/components/schemas/positive_currency",
-                            description: "Amount outstanding on all mortgages against this property",
-                            example: 999.99,
-                          },
-                          percentage_owned: {
-                            type: :number,
-                            format: :decimal,
-                            description: "Percentage share of the property which is owned by the applicant's partner",
-                            example: 99.99,
-                            minimum: 0,
-                            maximum: 100,
-                          },
-                          shared_with_housing_assoc: {
-                            type: :boolean,
-                            description: "Property is shared with a housing association",
-                          },
-                          subject_matter_of_dispute: {
-                            type: :boolean,
-                            description: "Property is the subject of a dispute",
-                          },
-                        },
-                      },
+                      items: { "$ref" => "#/components/schemas/Property" },
                     },
                     capitals: { "$ref" => "#/components/schemas/Capitals" },
                     vehicles: {
                       type: :array,
                       description: "One or more vehicles' details",
-                      items: {
-                        type: :object,
-                        required: %i[value date_of_purchase],
-                        properties: {
-                          value: {
-                            "$ref" => "#/components/schemas/positive_currency",
-                            description: "Financial value of the vehicle",
-                          },
-                          loan_amount_outstanding: {
-                            "$ref" => "#/components/schemas/currency",
-                            description: "Amount remaining, if any, of a loan used to purchase the vehicle",
-                          },
-                          date_of_purchase: {
-                            type: :string,
-                            format: :date,
```

Comment:
@starswan Using ` format: dates`  instead of ` format: date`  in swagger/v5/swagger.yaml